challenge3/webapp/masuri/app/controllers/houses_controller.rb

```diff
@@ -0,0 +1,25 @@
+class HousesController < ApplicationController
+  before_action :set_house, only: :show
+
+  def index
+    @houses = House.all
+    cities = @houses.pluck(:city).uniq
```

Comment:
Ruby で `uniq` するのではなく、データベースで処理する方法もある。

```suggestion
    cities = @houses.select(:city_id).distinct.pluck(:city)
```